fireblocks_sdk/sdk.py

```diff
@@ -1618,8 +1618,11 @@ def _post_request(self, path, body=None, idempotency_key=None):
         response = self.http_session.post(self.base_url + path, headers=headers, json=body, timeout=self.timeout)
         return handle_response(response)
 
-    def _put_request(self, path, body=None):
+    def _put_request(self, path, body=None, query_params=None):
         body = body or {}
+        query_params = query_params or {}
+        if query_params:
+            path = path + "?" + urllib.parse.urlencode(query_params)
```

Comment:
Why not to use `params` argument instead?
See docs: https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls

Reply:
You are right, but because of the jwt signing we have to build the full path with the query params for signature before the request is sent